app/models/events/rubygem_event.rb

```diff
@@ -59,6 +59,17 @@ class Events::RubygemEvent < ApplicationRecord
     attribute :owner_gid, :global_id
   end
 
+  OWNER_ROLE_UPDATED = define_event "rubygem:owner:updated" do
```

Comment:
should this now be...
```suggestion
  OWNER_ROLE_UPDATED = define_event "rubygem:owner:role_updated" do
```